File: EventCheckin.Api/Models/Events/EventDayModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace EventCheckin.Api.Models.Events
{
    public class EventDayModel
    {
        public int Id { get; set; }
        
        public int EventId { get; set; }

        public string EventDayName { get; set; }

        public DateTime? StartDate { get; set; }

        public DateTime? EndDate { get; set; }

        public EventModel EventModel { get; set; }
    }
}

File: EventCheckin.ClientApp/src/app/api/models/event-day-model.ts
/* tslint:disable */
/* eslint-disable */
import { EventModel } from '../models/event-model';
export interface EventDayModel {
  endDate?: string | null;
  eventDayName?: string | null;
  eventId?: number;
  eventModel?: EventModel;
  id?: number;
  startDate?: string | null;
}
